keyboards/lily58/keymaps/BTJ01/_keymap.c

```diff
@@ -0,0 +1,270 @@
+),                                   /*        |---Fn-SPACE---|___Fn-SPACE___|     */
+
+/* FnCaps **********************************************************************************************************************
+ * .-----------------------------------------.                               .-----------------------------------------.
+ * |TskMgr|      |      |      |      | RESET|                               | Break| NumLk| ScrLk| PScr |      |      |
+ * |------+------+------+------+------+------|                               |------+------+------+------+------+------|
+ * |      |      |      |      |      | DEBUG|                               |      | MNxt | VolUp| MPly |      |      |
+ * |------+------+------+------+------+------|                               |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |-------.               .-------|      | MPrv | VolDn| Mute |      |      |
+ * |------+------+------+------+--^^--+------|       |               |       |------+--^^--+------+------+------+------|
+ * |      |      |      |      |      |      |-------|               |-------|      |      |      |      |      |      |
+ * '-----------------------------------------/       /                \      \-----------------------------------------'
+ *                   |QWERTY|COLMAK|      | /       /                  \      \  |      |      |      |
+ *                   |      |      |      |/       /                    \      \ |      |      |      |
+ *                   '----------------------------'                      '------''--------------------'
+ */
+[_FC] = LAYOUT( \
+  C(S(KC_ESC)),XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, RESET,                       KC_BRK,  KC_NLCK, KC_SLCK, KC_PSCR, XXXXXXX, XXXXXXX, \
+  XXXXXXX,     XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, DEBUG,                       XXXXXXX, KC_MNXT, KC_VOLU, KC_MPLY, XXXXXXX, XXXXXXX, \
+  _______,     XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                     XXXXXXX, KC_MPRV, KC_VOLD, KC_MUTE, XXXXXXX, XXXXXXX, \
+  XXXXXXX,     XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, \
+                               QWRT, COLE,  XXXXXXX, XXXXXXX,            XXXXXXX,  XXXXXXX, XXXXXXX, XXXXXXX \
+  )
+};
+
+int RGB_current_mode;
+
+// Setting ADJUST layer RGB back to default
+void update_tri_layer_RGB(uint8_t layer1, uint8_t layer2, uint8_t layer3) {
+  if (IS_LAYER_ON(layer1) && IS_LAYER_ON(layer2)) {
+    layer_on(layer3);
+  } else {
+    layer_off(layer3);
+  }
+}
+
+void matrix_init_user(void) {
+    #ifdef RGBLIGHT_ENABLE
+      RGB_current_mode = rgblight_config.mode;
+    #endif
+    //SSD1306 OLED init, make sure to add #define SSD1306OLED in config.h
+    #ifdef SSD1306OLED
+        iota_gfx_init(!has_usb());   // turns on the display
+    #endif
+}
+
+//SSD1306 OLED update loop, make sure to add #define SSD1306OLED in config.h
+#ifdef SSD1306OLED
```

Comment:
In general, we recommend the OLED driver over the SSD1306OLED code. 

https://docs.qmk.fm/#/feature_oled_driver?id=ssd1306h-driver-conversion-guide